uploadPatchSet.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+FLOW="$1"
+ENVIRONMENT="$2"
+
+case $# in
+  2)
+    ENVIRONMENT="$2"
+    ;;
+  1)
+    ENVIRONMENT="local"
+    ;;
+  *)
+    echo "not enough arguments supplied.  You must supply the flowName to this command."
+    return 1
```

Comment:
When I ran this script with no arguments, the following was printed to the terminal:

```
not enough arguments supplied.  You must supply the flowName to this command.
/Users/tim/src/ihub-developer-scripts/uploadPatchSet.sh: line 14: return: can only `return' from a function or sourced script
overriding api
overriding flow
using token
Got unexpected HTTP response 404. This is likely an error.
```

When I changed line 14 to read: `exit 1`, the following was printed:

```
not enough arguments supplied.  You must supply the flowName to this command.
```

Reply:
fixed